.github/workflows/create-tag.yml

```diff
@@ -1,6 +1,6 @@
 ---
-## Workflow to create a new git tag if version.py variable version gets updated
-name: release
+# Workflow to create a new git tag if version.py variable version gets updated
+name: create-tag
```

Comment:
This will require a change in the infra part, so the terraform definition for the OIDC is configured for the `create-tag`


```suggestion
# If you change the name, please adjust the changes in https://github.com/elastic/oblt-infra/tree/main/conf/resources/repos/elastic-serverless-forwarder
name: create-tag
```